src/pages/workspace/workflows/approvals/WorkspaceWorkflowsApprovalsExpensesFromPage.tsx

```diff
@@ -150,18 +153,23 @@ function WorkspaceWorkflowsApprovalsExpensesFromPage({policy, isLoadingReportDat
         Navigation.goBack();
     }, [isInitialCreationFlow]);
 
-    const nextButton = useMemo(
-        () => (
+    const button = useMemo(() => {
+        let buttonText = isInitialCreationFlow ? translate('common.next') : translate('common.save');
+
+        if (shouldShowListEmptyContent) {
+            buttonText = translate('common.buttonConfirm');
+        }
+
+        return (
             <FormAlertWithSubmitButton
-                isDisabled={!selectedMembers.length}
-                buttonText={isInitialCreationFlow ? translate('common.next') : translate('common.save')}
-                onSubmit={nextStep}
+                isDisabled={!shouldShowListEmptyContent && !selectedMembers.length}
+                buttonText={buttonText}
+                onSubmit={shouldShowListEmptyContent ? () => Navigation.goBack() : nextStep}
```

Comment:
@blazejkustra Should apply this to approver list page

Reply:
Actually no, we should always use `nextStep` handler for approver list page. There is an edge case where you want to remove an approver who is in circular reference, you can change the code and try it 😄

Reply:
> There is an edge case where you want to remove an approver who is in circular reference

The circular reference isn't allowed on Newdot. So I am not sure your mean

Could you help to clear your edge case?

Reply:
@blazejkustra Could you check above comment?

Reply:
Sure, let me post a video in a sec

Reply:
In the editor you can create a workflow with circular reference (you can't save it, but you can get to this state). 

![image](https://github.com/user-attachments/assets/5cfd6577-5900-4a78-bff2-40ac63280cde)

1. Try to remove this duplicated approver and make sure there are no other approvers to choose from:

![image](https://github.com/user-attachments/assets/de2a16e2-8ed3-426a-b572-7b2383bcbd3e)

2. Once your approver is unchecked, empty state is displayed: 

![image](https://github.com/user-attachments/assets/1133ec33-2c16-4056-848e-fa4fe8f734a7)
 
3. Clicking on got it has to trigger `nextStep` function in oder to call `Workflow.clearApprovalWorkflowApprover(approverIndex)` function

Here is the video:

https://github.com/user-attachments/assets/78973d06-4601-415f-b79c-d4d9ca888b54

Reply:
Thanks for your explanation